mobile/ios/device/ios-core.ts

```diff
@@ -1197,7 +1204,7 @@ export class GDBServer implements Mobile.IGDBServer {
 						this.sendCore(this.encodeData("D"));
 					}
 				} else {
-					this.socket.pipe(new GDBStandardOutputAdapter()).pipe(process.stdout);
+					this.socket.pipe($injector.resolve(GDBStandardOutputAdapter, { deviceIdentifier: this.deviceIdentifier }));
```

Comment:
it's better to use the current instance of the injector instead of the global one:
`$injector` -> `this.$injector`